bcda/client/bluebutton.go

```diff
@@ -66,7 +64,7 @@ type APIClient interface {
 	GetExplanationOfBenefit(jobData models.JobEnqueueArgs, patientID string, claimsWindow ClaimsWindow) (*fhirModels.Bundle, error)
 	GetPatient(jobData models.JobEnqueueArgs, patientID string) (*fhirModels.Bundle, error)
 	GetCoverage(jobData models.JobEnqueueArgs, beneficiaryID string) (*fhirModels.Bundle, error)
-	GetPatientByIdentifierHash(jobData models.JobEnqueueArgs, hashedIdentifier string) (string, error)
+	GetPatientByMbi(jobData models.JobEnqueueArgs, mbi string) (string, error)
```

Comment:
Any argument to keep `GetPatientByIdentifierHash` functionality?  Or is this a known permanent change?